apps/ex_wire/lib/ex_wire/handshake/handshake.ex

```diff
@@ -327,4 +327,9 @@ defmodule ExWire.Handshake do
         {:invalid, reason}
     end
   end
+
+  @spec new_nonce() :: binary()
+  def new_nonce do
```

Comment:
Extracted this into a function (a public one) because I'm having to use this from other places -- mostly tests. I could simply use `ExthCrypto.Math.nonce/1` from the tests, but since the value of `@nonce_len` lives in this module, having a random `32` in the tests can lead to confusion. So I think it best to have a function in this module that generates a nonce of the appropriate length for a handshake.